etc/behat/suites/ui-cart.yml

```diff
@@ -21,7 +21,7 @@ default:
                 - Sylius\Behat\Context\Setup\PaymentContext:
                     paymentMethodRepository: "@sylius.repository.payment_method"
                     sharedStorage: "@sylius.behat.shared_storage"
-                    paymentMethodFactory: "@sylius.factory.payment_method"
+                    paymentMethodFactory: "@sylius.behat.factory.payment_method"
```

Comment:
I have not seen it (the class) yet, but I think we could prefix it with ``test`` instead. Unless there is something really specific to Behat there.